educator/src/Dscp/Educator/Web/Educator/Auth.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE TypeOperators #-}
+-- | Nessesary types and implementation for Educator authenthication
```

Comment:
Necessary.

Reply:
I feel ashamed.

Reply:
Don't be, @pva701 does it even worse :peka: